app/controllers/api/v1/reservations.rb

```diff
@@ -0,0 +1,43 @@
+class Api::V1::ReservationsController < ApplicationController
+  before_action :set_reservation, only: %i[create]
+  before_action :set_user, only: %i[update destroy]
+  before_action :authorized
+
+  def index
+    @reservations = if current_user.admin == true
+                      Reservation.all
+                    else
+                      current_user.reservations
+                    end
+    render json: { message: 'Showing Reservations', reservations: @reservations }, status: :ok
+  end
```

Comment:
- Since this application is not using devise, `current_user` is not a valid function. It will be possible to send the user id from the frontend form. Please update your index route to use the user id in the params instead of current_user
- I see you tried to distinguish between admins and normal users. Nice one👍. However, our application won't have admins, so this distinction may be unnecessary.